src/main.rs

```diff
@@ -134,17 +134,17 @@ fn parse_args() -> PbftCliArgs {
          "connection endpoint for validator (default 'tcp://localhost:5050')")
         (@arg verbose: -v --verbose +multiple
          "increase output verbosity")
-        (@arg logconfig: -L --log_config +takes_value
+        (@arg logconfig: -L --log-config +takes_value
          "path to logging config file")
-        (@arg exponential_retry_base: -b --exponential_retry_base +takes_value
+        (@arg exponential_retry_base: -b --exponential-retry-base +takes_value
```

Comment:
While, this compiles, these need to be wrapped like so: `--("exponential-retry-base")`

Reply:
Fixed